app/controllers/api/subcollections/generic_objects.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module Subcollections
+    module GenericObjects
+      def generic_objects_query_resource(object_definition)
+        object_definition.generic_objects
```

Comment:
what if the generic object definition has no generic object, is this nil here ? what is returned by API ?

Reply:
@abellotti if it has no generic objects, it just returns an empty array, so it just returns `"resources": []`